GithubPractice/ViewController.swift

```diff
@@ -27,10 +29,19 @@ class ViewController: UIViewController, UITextFieldDelegate {
         
         saveButton.layer.cornerRadius = 8
         
+        if let job = defaults.string(forKey: "Job") {
+            jobPositionField.text = job
+        }
+        
+        if let date = defaults.object(forKey: "BirthDay"){
+            birthdayPicker.date = date as! Date
```

Comment:
as an alternative u can try to store TimeInterval instead of Date.

Reply:
I found on web that tTimeInterval is used when we need to store data as hours, minutes or day separately, in this case I think we should let this work as it is)

Reply:
avoid force casting. instead check for value like:
        if let date = defaults.object(forKey: birthDayKey) as? Date {
            birthdayPicker.date = date
        }

Reply:
> avoid force casting. instead check for value like:
> if let date = defaults.object(forKey: birthDayKey) as? Date {
> birthdayPicker.date = date
> }

done